packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -385,6 +385,13 @@ test.serial('distributes fees per BLD staker decision', async t => {
 
   const ContractFee = 302000n; // see split above
   t.is(((ContractFee - 250000n) * 5n) / 10n, 26000n);
+
+  if (defaultManagerType === 'xs-worker') {
+    // XXX for some reason the code after this when run under XS fails with:
+    // message: 'unsettled value for "kp2526"',
```

Comment:
We have an issue now which suffices and has the advantage of an easy query for cleanup.
```suggestion
    // UNTIL https://github.com/Agoric/agoric-sdk/issues/10847
```

Reply:
Just removed all this code because of https://github.com/Agoric/agoric-sdk/commit/cab3fbe1f1f01c484da0a745830d2e85513c2d44